projects/packages/forms/src/blocks/contact-form/child-blocks.js

```diff
@@ -380,6 +385,32 @@ export const childBlocks = [
 			},
 		},
 	},
+	{
+		name: 'field-number',
+		settings: {
+			...FieldDefaults,
+			title: __( 'Number Input Field', 'jetpack-forms' ),
+			description: __( 'Collect numbers from site visitors.', 'jetpack-forms' ),
+			icon: renderMaterialIcon(
+				<Path
+					fill={ getIconColor() }
+					d="M12 7H4V8.5H12V7ZM19.75 17.25V10.75H4.25V17.25H19.75ZM5.75 15.75V12.25H18.25V15.75H5.75Z"
```

Comment:
on a follow up PR we should probably design a different icon?

Reply:
Yes! :-)